core/fullHistory/nilHistoryProcessor.go

```diff
@@ -4,37 +4,37 @@ import (
 	"errors"
 )
 
-type nilHistoryProcessor struct {
+type nilHistoryRepository struct {
 }
 
-// NewNilHistoryProcessor -
-func NewNilHistoryProcessor() (*nilHistoryProcessor, error) {
-	return new(nilHistoryProcessor), nil
+// NewNilHistoryProcessor returns a not implemented history repository
+func NewNilHistoryProcessor() (*nilHistoryRepository, error) {
+	return new(nilHistoryRepository), nil
 }
 
 var errNilHistoryProcessorImplementation = errors.New("this a nil implementation of history processor")
 
-// PutTransactionsData -
-func (nhs *nilHistoryProcessor) PutTransactionsData(_ *HistoryTransactionsData) error {
+// PutTransactionsData returns a not implemented error
```

Comment:
`errNilHistoryProcessorImplementation` - typo in the error message.

Reply:
done.